.github/workflows/qe-image.yml

```diff
@@ -0,0 +1,52 @@
+name: Push qe images for new tag
+on:
+  push:
+    tags:
+      - '*'
+env:
+  IMAGE_REGISTRY: quay.io
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+        go:
+          - '1.21'
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - name: Log in to Quay.io
+        uses: redhat-actions/podman-login@v1
+        with:
+          username: ${{ secrets.QUAY_IO_USERNAME }}
+          password: ${{ secrets.QUAY_IO_PASSWORD }}
+          registry: ${{ env.IMAGE_REGISTRY }}
+      - name: create script of building and push images
+        run: |
+          cat <<EOF > build-qe-images.sh
+          OS=linux ARCH=amd64 CRC_INTEGRATION_IMG_VERSION=v${1} make containerized_integration
```

Comment:
We are now including the linux/support so can we add it here as well?

Reply:
@adrianriobo Sorry, I don't get it. Could you explain more about it?

Reply:
ahaha indeed you don get it I wrote it wrong /0\ , sorry

I meant we are adding support for linux/arm64 previously we were not building those images but now we will need them.. So can you add that tuple to the matrix values: (linux,arm64)

Reply:
just saw your update on the matrix so basically is now remove the excluded on that tuple (linux,arm64)

Reply:
updated